lib/migrate_ast.ml

```diff
@@ -31,39 +31,39 @@ module Asttypes_ = Asttypes
 module Parsetree_ = Parsetree
 
 include (
-  Migrate_parsetree.Ast_406 :
+  Migrate_parsetree.Ast_408 :
   module type of struct
-      include Migrate_parsetree.Ast_406
+      include Migrate_parsetree.Ast_408
     end
-    with module Location := Migrate_parsetree.Ast_406.Location
-     and module Outcometree := Migrate_parsetree.Ast_406.Outcometree
-     and module Asttypes := Migrate_parsetree.Ast_406.Asttypes
-     and module Ast_helper := Migrate_parsetree.Ast_406.Ast_helper
-     and module Parsetree := Migrate_parsetree.Ast_406.Parsetree
+    with module Location := Migrate_parsetree.Ast_408.Location
+     and module Outcometree := Migrate_parsetree.Ast_408.Outcometree
+     and module Asttypes := Migrate_parsetree.Ast_408.Asttypes
+     and module Ast_helper := Migrate_parsetree.Ast_408.Ast_helper
+     and module Parsetree := Migrate_parsetree.Ast_408.Parsetree
  )
 
-module Asttypes = Migrate_parsetree.Ast_406.Asttypes
-module Ast_helper = Migrate_parsetree.Ast_406.Ast_helper
-module Parsetree = Migrate_parsetree.Ast_406.Parsetree
+module Asttypes = Migrate_parsetree.Ast_408.Asttypes
+module Ast_helper = Migrate_parsetree.Ast_408.Ast_helper
+module Parsetree = Migrate_parsetree.Ast_408.Parsetree
```

Comment:
These 3 lines are not necessary.

Reply:
It doesn't seem to compile without it, so I'm keeping it.

Reply:
Yes, it's only working in 4.08